subsystems/sidecar/index.js

```diff
@@ -988,6 +936,15 @@ class Sidecar extends ReadyResource {
     }
   }
 
+  _generateAppStorage (bundleLink) {
+    const pathMatcher = isWindows ? /[/\\]/ : /\//
```

Comment:
we have a link module, use instead to check if its a pear link or do startsWith(pear://) if you wanna yolo it

Reply:
(For context see the second review comment)

we're not receiving pear:// here it seems, and got a cursed Buffer value too

```
➜  pear git:(hyperdb-store-refactor) ✗ npm test -- --log | grep "bundleLink"
bundleLink: jiec6je9w3f83jpe716x6xr1pbaexna34wube74se96y7pr5ziwy
bundleLink: w8nya7wsh3y3wdc6gj1ixrfcjidaex81gotxzx3dnzhocxp4mjio
bundleLink: 3inx7kusgdqdqphg7egmtnjtbbny5ik7xewdhhzduws7u7mdpexo
bundleLink: 6mkf5bit8a4izfp8sbb6qb4ymba96twwaf1tjyhrrj8mwxoz616o
bundleLink: frkjc7weodaggjur5u3b5p7dsrgfdncfcemchwdb4y8c4z48ccao
bundleLink: t9funascht4kt4a7cu1fin7h4utr3ain6rbmdbksy6hsd1ntyhjy
bundleLink: dxg9pi91i46jfgs5gzjx1wnei7ht54ta71en3ec45nmhrauqm6qy
bundleLink: 8itayum84yu1ynutd85u5jn4hpfr3h5w7c93b4crciri5n5ze5yo
bundleLink: grti9ug4npemjot8zn1gnw4c83j8xtykm6h8tdm1z4w4aho9ckoy
bundleLink: tqa7jj1fnbd4xh5k7w8omhbehwgoxdyz7qajciy6mjcgenrhr17o
bundleLink: gytt3syy53agi1mbm4jk8nue6gtwutmqawtdf5udjmqndw97wdto
bundleLink: zyxwijrc45u3ncgy1qow4b5dtb65imx5wjuebwgtqmjpe6da37ho
bundleLink: 74qcjnme51fq36qtdjrfnp5h4qeeccw8pjhrqaw3bupnrhb8pyzy
bundleLink: etu1zy3z3xsoy8ggb4d6f5d45cxx6m8fa56cpnmwjh85h4ei3o1y
bundleLink: rXJ���e䄗ƻcX@f����Oȋ�<��
bundleLink: D�b>Ot3mP�\��
                         �I�Z���)媻V?�6V
bundleLink: 3ueddhg5pnxg8chiz3s6hn6npe18pzw45ns8n791uqargdg6swyo
bundleLink: tkfenmh4rsurchc1c4eh6sum3dz7yn7rchtuorup5ygwa9aztypo
bundleLink: 81m3upqp913iut1isoey4564qapwiobpatar74ah4tthyxcka98y
bundleLink: 9cju6imed9pph6thfurxz4takocje4sxomt6ax73rtfztrjnmhuy
bundleLink: 15nijtpobtjjfccj5dyxzh1fotjqyxecdazsrc6qgkkqhkrmr5xo
```